src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -554,7 +555,7 @@ public void testUserApiForNonSuperAdmin() throws Exception {
 
         // Put hidden users
         response = rh.executePutRequest("/_opendistro/_security/api/internalusers/hide", "[{ \"op\": \"add\", \"path\": \"/sarek/description\", \"value\": \"foo\" }]", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
```

Comment:
Why is this bad request ? it is updating reserved user and password field is not present

Reply:
Ditto

Reply:
Fixed the test.